lib/utils/action-hook-delay.js

```diff
@@ -163,11 +164,16 @@ class ActionHookDelayProcessor extends Emitter {
     this._retryCount++;
   }
 
-  _onNoResponseGiveUpTimer() {
+  async _onNoResponseGiveUpTimer() {
```

Comment:
this no longer needs to be declared as async, does it?

Reply:
> this no longer needs to be declared as async, does it?

Yes you are right @davehorton  , removed async